lib/OpenQA/Script/CloneJob.pm

```diff
@@ -175,7 +176,7 @@ sub create_url_handler {
     my $local = OpenQA::Client->new(
         api       => $local_url->host,
         apikey    => $options->{'apikey'},
-        apisecret => $options->{'apisecret'});
+        apisecret => $options->{'apisecret'})->ioloop(Mojo::IOLoop->singleton);
```

Comment:
could you please explain this change a bit more in git commit message? I read "Use IO loops and die to allow testing" but sorry I don't understand if there could be any side-effects or what this actually does now.

Reply:
This sets the IO loop of the underlying `Mojo::UserAgent`. I guess the usual test setup causes a different IO loop to be used and this basically "ensures the default". It would be indeed useful to know why this is required here.